source/regions/quick-start.rst

```diff
@@ -108,6 +107,9 @@ Regions can overlap in WorldGuard, which provides a lot of flexibility but can b
 
         /rg flag mine build allow
 
+.. warning::
+    The ``build`` flag has some additional properties and we generally recommend not to change it, unless you know what you are doing. See the :doc:`region flag overview <flags>` for more information.
+
```

Comment:
I *feel* like this is one of the rare valid use cases of the build-flag, but I'm still not sure if it's a good idea to tell people to change it here, so I added an additional warning